torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -154,14 +207,22 @@ def _invoke_subgraph_placeholder_wrapper(func, args):
     return func(*args, **kwargs)
 
 
-def mark_compile_region(fn=None):
+def mark_compile_region(
+    fn=None, aot_config: Optional[NestedCompileRegionOptions] = None
+):
     """
     This wrapper instructs torch.compile to compile the wrapped region once and
     reuse the compiled artifact, instead of the usual way of aggressively
     inlining the function.
 
     Under the hood, it tells TorchDynamo to use InvokeSubgraph HOP for the
     region. For PyTorch eager, this is a no-op.
+
+    Args:
+        fn: The function to wrap
+        aot_config: Optional config to use for compiling the subgraph.
+            Warning: this is an experimental feature under development and
+            not ready for use yet.
```

Comment:
Maybe just call this `options` at this point. It's not an AOTConfig (that's something else)